tests/unit/info.tcl

```diff
@@ -274,5 +274,11 @@ start_server {tags {"info" "external:skip"}} {
             $rd close
         }
 
+        test {stats: cmd duration} {
+            r config resetstat
+            regexp "\r\neventloop_duration_cmd_sum:(.*?)\r\n" [r info stats] -> cmd_duration
```

Comment:
why not use `[s eventloop_duration_cmd_sum]`?

Reply:
I am not so familiar with this TCL test framework... I forgot about this proc.